addon/components/o-s-s/url-input.stories.js

```diff
@@ -0,0 +1,118 @@
+import hbs from 'htmlbars-inline-precompile';
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'Components/OSS::UrlInput',
+  component: 'url-input',
+  argTypes: {
+    value: {
+      description: 'Value of the input',
+      table: {
+        type: {
+          summary: 'string'
+        },
+        defaultValue: { summary: undefined }
+      },
+      control: { type: 'text' }
+    },
+    disabled: {
+      description: 'Disable the default input (when not passing an input named block)',
+      table: {
+        type: {
+          summary: 'boolean'
+        },
+        defaultValue: { summary: false }
+      },
+      control: { type: 'boolean' }
+    },
+    placeholder: {
+      description: 'Placeholder of the input',
+      table: {
+        type: {
+          summary: 'string'
+        },
+        defaultValue: { summary: undefined }
+      },
+      control: { type: 'text' }
+    },
+    errorMessage: {
+      description: 'An error message that will be displayed below the input-group.',
+      table: {
+        type: {
+          summary: 'string'
+        },
+        defaultValue: { summary: false }
+      },
+      control: { type: 'text' }
+    },
+    onChange: {
+      type: { name: 'Method' },
+      description: 'Method called every time the input is updated',
+      table: {
+        type: {
+          summary: 'onChange(value: string): void'
```

Comment:
```suggestion
          summary: 'onChange(value: string, isValid: boolean): void'
```
It is the declaration in the ts file